stetho-sample/build.gradle

```diff
@@ -31,6 +31,7 @@ android {
 dependencies {
     debugCompile project(':stetho')
     debugCompile project(':stetho-urlconnection')
+    debugCompile project(':stetho-js-rhino')
```

Comment:
I am sorry. That one slipped through.
On Aug 4, 2015 6:04 PM, "Josh Guilfoyle" <notifications@github.com> wrote:

> In stetho-sample/build.gradle
> <https://github.com/facebook/stetho/pull/245#discussion_r36206448>:
>
> > @@ -31,6 +31,7 @@ android {
> >  dependencies {
> >      debugCompile project(':stetho')
> >      debugCompile project(':stetho-urlconnection')
> > +    debugCompile project(':stetho-js-rhino')
>
> Oops, this should not have been committed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/stetho/pull/245/files#r36206448>.
>